app/assets/v2/js/grants/detail.js

```diff
@@ -60,15 +60,21 @@ $(document).ready(function() {
       let deployedSubscription = new web3.eth.Contract(compiledSubscription.abi, data.contract_address);
 
       web3.eth.getAccounts(function(err, accounts) {
-
-
-        deployedSubscription.methods.endContract().send({from: accounts[0], gasPrice: 4000000000})
+        deployedSubscription.methods.endContract()
+          .send({from: accounts[0], gasPrice: 4000000000})
+          // .on('transactionHash', function() {
+          //   // Waiting State screen
```

Comment:
is this intended to be commented out?  if so can we remove it?

Reply:
I'm not sure if that is the right way to add it , which is why I tagged @captnseagraves to calrify on that ! 
This is essentially when to load waiting screen -> once user clicks on cancel grant 
would you have context on this @owocki  ?

Reply:
@owocki @thelostone-mc Yes, this is how we should implement the waiting screen. We can uncomment this code.

Reply:
👍